lms/envs/common.py

```diff
@@ -4964,3 +4964,11 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 # The expected value is an Integer representing the cutoff point (in months) for inclusion to the message. Example:
 # a value of `3` would include learners who have logged in within the past 3 months.
 BULK_COURSE_EMAIL_LAST_LOGIN_ELIGIBILITY_PERIOD = None
+# HTML tags allowed within the body of a message authored via the Bulk Course Email Tool
+BULK_COURSE_EMAIL_ALLOWED_HTML_TAGS = [
+    "a", "abbr", "address", "area", "article", "aside", "b", "blockquote", "br", "caption", "cite", "code", "col",
+    "colgroup", "dd", "del", "div", "dl", "dt", "em", "embed", "figcaption", "figure", "footer", "h1", "h2", "h3",
+    "h4", "h5", "h6", "h7", "header", "hr", "i", "img", "ins", "li", "link", "main", "map", "mark", "meta", "object",
+    "ol", "p", "param", "pre", "q", "s", "samp", "section", "small", "source", "span", "strong", "style", "sub", "sup",
+    "table", "tbody", "td", "textarea", "th", "thead", "tr", "u", "ul", "video", "wbr",
+]
```

Comment:
This list is based off historical data of content used within emails by edX partners in the past. I scraped a list of unique element tags in our saved email course email data and used this to come up with a sane list of default tags to allow.

Messages sent by course teams to learners contain a variety of learning content and I wanted to make sure the tool is still flexible and we aren't stripping too much content.

We _are_ trying to keep people from embedding forms and other potentially dangerous data (and then storing that in our database).

Reply:
Have you talked at all to Jeff Witt about the a11y impact of these choices? This might be something he would be interested in, because its one of the moments we can control the accessibility of the content used by course teams.

Reply:
@connorhaugh I haven't! The thought hadn't crossed my mind. I'll reach out.